client/app/utils/transitImmutableConverter.js

```diff
@@ -31,6 +31,7 @@ const createReader = function createReader() {
     handlers: {
       ':': (rep) => `:${rep}`,
       list: (rep) => Immutable.List(rep).asImmutable(),
+      u: (rep) => rep.toString(),
```

Comment:
This was a nasty surprise: Immutable maps didn't do well with UUID objects as keys, so I converted them to string representation.